index.js

```diff
@@ -1,5 +1,8 @@
 "use strict";
 
+const traverse = require('babel-traverse').default;
```

Comment:
Ah, it didn't, but I added support in https://github.com/prettier/prettier/pull/1637/commits/cbc1929c64db558b4e444500bca3d2ce1d550359